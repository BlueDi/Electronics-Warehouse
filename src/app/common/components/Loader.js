import React, { Component } from 'react';
import { Dimmer, Loader } from 'semantic-ui-react';

class WHLoader extends Component {
  render() {
    return (
      <Dimmer active inverted>
        <Loader indeterminate inverted>
          {this.props.text}
        </Loader>
      </Dimmer>
    );
  }
}

export default WHLoader;
