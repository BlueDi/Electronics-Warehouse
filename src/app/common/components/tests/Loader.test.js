import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../Loader';

test('Loader Render', () => {
  const component = renderer.create(<Loader />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
