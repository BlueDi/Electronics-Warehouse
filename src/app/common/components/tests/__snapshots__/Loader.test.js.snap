// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Render 1`] = `
<div
  className="ui active transition visible inverted dimmer"
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="ui indeterminate inverted loader"
    />
  </div>
</div>
`;
